ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103788_edit-deferred-decision.js

```diff
@@ -0,0 +1,343 @@
+/// <reference types ='Cypress'/>
+
+// uri to be updated once academisation API is integrated
+let url = Cypress.env('url') + '/task-list/2054?rd=true'
+let projecList = Cypress.env('url') + '/project-list'
+
+describe(' Edit Deferred journey', () => {
+    beforeEach(() => {
+        cy.clearCookies()
+        cy.visit(url)
+    })
+
+// Edit Deferred Path - Regional Director, Additional information needed 
+it('TC01: J3 Edit a recorded decision Deferred - Reg Director Region, Finance', () => {
+    // Click on change your decision button 
+    cy.changeDecision().should('contain.text', 'Change your decision').click()
+     //select iniital decision
+    cy.deferredRadioBtn().click()
+    // clicks on the continue button
+    cy.continueBtn().click()
+    cy.get('[id="regionaldirectorforregion-radio"]').click()
+    // clicks on the continue button
+    cy.continueBtn().click()
+    cy.addInfoNeededBox()
+    .invoke('attr', 'aria-expanded')
+    .then(ariaExpand => {
+        if (ariaExpand.includes(true)) {
+            // clicks on finance box
+                // clicks on the Give Reasons box
+            cy.addInfoNeededText().clear().type('This is the second test')
```

Comment:
It might help with troubleshooting and assertions if this content said something relating to regionaldirectorforregion.